CHANGELOG.md

```diff
@@ -1,5 +1,439 @@
 # Change Log
 
+## [v2.2.0-develop-2019061023](https://github.com/swift-nav/piksi_tools/tree/v2.2.0-develop-2019061023) (2019-06-10)
```

Comment:
v2.3.17?

Reply:
Never mind, I guess this changelog script pulls in all of the dev branches too?

Reply:
yes but I'll manually delete this top part as it is confusing for what we are trying to document